packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -327,6 +339,16 @@ const makeRenderer = (
 		return routesPromise.then(resolvedRoutes =>
 			initializeStore(resolvedRoutes).then(store => {
 				if ('skeleton' in request.query) {
+					// TODO: what is the skeleton? Do we need to also check the ld flags for css styles here too?
```

Comment:
cc @mmcgahan --- I can't remember the purpose of `skeleton`. Do we need to include the LD new style conditions here as well, or should I remove?

Reply:
actually, if you could just remove the whole `if ('skeleton')` section, that's probably best - it's a vestige of some offline-first support that we experimented with early on, but it never took root so we should roll it back